src/libs/tools/include/errors/baseNotification.hpp

```diff
@@ -0,0 +1,94 @@
+#ifndef ELEKTRA_BASENOTIFICATION_HPP
+#define ELEKTRA_BASENOTIFICATION_HPP
+
+#include <string>
+#include <key.hpp>
+#include <utility>
+#include <kdbtypes.h> // for kdb-types, code and description constants
+
+namespace kdb
+{
+namespace tools
+{
+namespace errors
+{
+/* common abstract class for warnings and errors */
+/* Because warning and errors share the same data members, a method can accept a ErrBase argument and the caller
```

Comment:
ErrBase seems to be a leftover?

Reply:
Your assumption is correct, the class got renamed to `BaseNotification`.
Comment gets fixed by commit c2ec7ebbb6bbddfda4f82b9d9e44aee27d9971e2.